src/fontra/views/editor/panel-reference-font.js

```diff
@@ -3,11 +3,20 @@ import Panel from "./panel.js";
 import * as html from "/core/unlit.js";
 
 import { ObservableController } from "/core/observable-object.js";
-import { div, input, label } from "/core/unlit.js";
-import { fileNameExtension, withTimeout } from "/core/utils.js";
+import { div, input, label, option, select } from "/core/unlit.js";
+import { fetchJSON, fileNameExtension, withTimeout } from "/core/utils.js";
 import { dialog } from "/web-components/modal-dialog.js";
 import { UIList } from "/web-components/ui-list.js";
 
+// prettier-ignore
```

Comment:
What does prettier do with it?

Reply:
Prettier lift downs these imports to after the line that the "Font" is imported. lib-font wants them to be imported before the main module is imported.

Reply:
Ahh, that's nasty. Thanks!

Reply:
Please add a comment to that effect.